CHANGELOG.md

```diff
@@ -1,3 +1,61 @@
+CHANGES IN VERSION 1.18.0 (FROM 1.17.1)
```

Comment:
important fixes that affect all drivers: 
- https://github.com/appium/appium-base-driver/commit/d946ba1846b5dd43c5ce59b244cf4eb5b55c7369
- https://github.com/appium/appium-base-driver/commit/9015d4c80d12d2ed62cd3f96b3389963f1d43349